packages/react-urql/examples/1-getting-started/package.json

```diff
@@ -1,37 +1,24 @@
 {
-  "name": "@urql/example-getting-started",
+  "name": "1-getting-started",
   "version": "1.0.0",
```

Comment:
Let's make sure in a follow-up PR to set `"private"` back to `true` please 🙏 Shouldn't affect CodeSandbox / the example itself, but since this isn't published it's safer to set this